core/src/main/java/org/apache/calcite/runtime/CalciteResource.java

```diff
@@ -70,7 +70,8 @@
   @BaseMessage("Illegal expression. Was expecting \"(DATETIME - DATETIME) INTERVALQUALIFIER\"")
   ExInst<CalciteException> illegalMinusDate();
 
-  @BaseMessage("Illegal overlaps expression. Was expecting expression on the form \"(DATETIME, EXPRESSION) OVERLAPS (DATETIME, EXPRESSION)\"")
+  @BaseMessage("Illegal overlaps expression. Was expecting expression on the form \"(DATETIME, "
```

Comment:
I assume some sort of formatter ran in this file, and there are no actual changes. If there is a change, let me know.